src/DataCollection.UWP/Controls/DateTimeEditor.xaml

```diff
@@ -0,0 +1,101 @@
+﻿<UserControl x:Class="Esri.ArcGISRuntime.OpenSourceApps.DataCollection.UWP.Controls.DateTimeEditor"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:ext="using:Microsoft.Toolkit.Uwp.UI.Extensions"
+             xmlns:local="using:Esri.ArcGISRuntime.OpenSourceApps.DataCollection.UWP.Controls">
+    <UserControl.Resources>
+        <Style x:Key="InsetButtonStyle"
+               BasedOn="{StaticResource ButtonBaseStyle}"
+               TargetType="Button">
+            <Setter Property="Width" Value="36" />
+            <Setter Property="Height" Value="36" />
+            <Setter Property="Margin" Value="0,1,1,1" />
+            <Setter Property="Padding" Value="0" />
+            <Setter Property="BorderThickness" Value="0" />
+            <Setter Property="HorizontalAlignment" Value="Right" />
+            <Setter Property="VerticalAlignment" Value="Center" />
+            <Setter Property="IsTabStop" Value="False" />
+        </Style>
+        <Style x:Key="InsetButtonPathIcon"
+               BasedOn="{StaticResource PathIconStyle}"
+               TargetType="Path">
+            <Setter Property="Width" Value="12" />
+            <Setter Property="Height" Value="12" />
+        </Style>
+        <Style x:Key="ComplexTexEditorStyle"
+               BasedOn="{StaticResource FieldEditorTextBoxStyle}"
+               TargetType="TextBox">
+            <Setter Property="AcceptsReturn" Value="False" />
+            <Setter Property="TextWrapping" Value="NoWrap" />
+        </Style>
+    </UserControl.Resources>
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="38" />
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="38" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+
+        <TextBox x:Name="DatePartTextBox"
+                 GotFocus="TextBox_GotFocus"
+                 LostFocus="DatePartTextBox_TextChanged"
+                 PlaceholderText="{x:Bind Converter={StaticResource LC}, ConverterParameter=DTE_DatePart_Placeholder}"
```

Comment:
LC could be a longer name like LocConverter. Up to you.

Reply:
I'm leaving as-is, only because that name wasn't introduced in this PR.